src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingMethodSymbol.cs

```diff
@@ -219,6 +222,42 @@ public override ImmutableArray<CSharpAttributeData> GetReturnTypeAttributes()
             return this.RetargetingTranslator.GetRetargetedAttributes(_underlyingMethod.GetReturnTypeAttributes(), ref _lazyReturnTypeCustomAttributes);
         }
 
+#nullable enable
+        internal override UnmanagedCallersOnlyAttributeData? UnmanagedCallersOnlyAttributeData
+        {
+            get
+            {
+                if (ReferenceEquals(_lazyUnmanagedAttributeData, UnmanagedCallersOnlyAttributeData.Uninitialized))
+                {
+                    var data = _underlyingMethod.UnmanagedCallersOnlyAttributeData;
+                    if (ReferenceEquals(data, UnmanagedCallersOnlyAttributeData.Uninitialized)
+                        || ReferenceEquals(data, UnmanagedCallersOnlyAttributeData.AttributePresentDataNotBound))
```

Comment:
>|| ReferenceEquals(data, UnmanagedCallersOnlyAttributeData.AttributePresentDataNotBound) [](start = 24, length = 88)

Is this check needed?

Reply:
Yes.

---
In reply to: [483714510](https://github.com/dotnet/roslyn/pull/47311#discussion_r483714510) [](ancestors = 483714510)